prime-dotnet-webapi/Services/LogService.cs

```diff
@@ -0,0 +1,29 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Prime.Models;
+using Prime.ViewModels;
+
+namespace Prime.Services
+{
+    public class LogService : BaseService, ILogService
```

Comment:
Perhaps we should name this something other than LogService so it doesn't get confused with a generic logger that the backend uses. Perhaps `ClientLogService`?

Reply:
Would `FrontEndLogService` be better because in case the Angular client isn't the only client of PRIME's web-api?